.drone/config.star

```diff
@@ -67,6 +67,8 @@ def main(ctx):
 
     steps.append(test_step("smoke_tests"))
     steps.append(test_step("cf_acceptance_tests"))
+    steps.append(test_step("kubecf_redeploy_cats_internetless"))
+    steps.append(test_step("cf_acceptance_tests"))
```

Comment:
Got an error for this line: https://kubecf-drone-ci-server.herokuapp.com/SUSE/kubecf/62.
`default: Linter: duplicate step name`.
The `test_step` does some magic on the name and path. You can try something like:
```python
def test_step(name, subset_name = ""):
    return step(
        name = "test:{name}{subset_name}".format(
            name = name,
            subset_name = ":{}".format(subset_name) if subset_name != "" else ""
        ),
        commands = [
            "{base_path}/steps/test/{name}.sh".format(
                base_path = DRONE_BASE_PATH,
                name = name,
            ),
        ],
        extra_volumes = [
            kube_volume(),
        ],
        network_mode = "host",
    )
```
Then use the `subset_name` arg with the `internetless` value.

Reply:
Resolved with commit 08d087cd6c60e6a46c1475aad1731ea05e188c61